packages/react-components/src/link/tests/chromatic/createTextLinkTestSuite.jsx

```diff
@@ -120,17 +120,16 @@ export function createTextLinkTestSuite(element, stories) {
                 </div>
             </Stack>
         )
-        .add("solid", () =>
-            <Inline verticalAlign="end">
-                <TextLink underline="solid" size="sm" href="#" element={element}>Flight details</TextLink>
-                <TextLink underline="solid" href="#" element={element}>Flight details</TextLink>
-            </Inline>
-        )
```

Comment:
I would keep this test, if we switch our default to something else, we would lose the solid test

Reply:
missed that will add it now